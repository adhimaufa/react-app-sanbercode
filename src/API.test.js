import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import CunsumeAPI from './Components/consumeAPI';
import ConsumeAPI from './Components/consumeAPI';

test(' api', () => {
  const component = renderer.create(
     <ConsumeAPI />
  );

  let tree = component.toJSON;
  expect(tree).toMatchSnapshot();

  // component.getInstance().handleNewTodo();
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});