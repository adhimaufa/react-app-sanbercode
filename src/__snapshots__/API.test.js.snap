// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` api 1`] = `[Function]`;
